assets/js/lib/api/wanda.js

```diff
@@ -1,16 +1,13 @@
 import axios from 'axios';
+import { networkClient } from '@lib/network'
 
 const baseURL = process.env.WANDA_URL;
 
-export const wandaClient = axios.create({
-  baseURL,
-});
-
 export const getExecutionResult = (executionID) =>
-  wandaClient.get(`/api/checks/executions/${executionID}`);
+  networkClient.get(`/api/checks/executions/${executionID}`, { baseURL: baseURL });
```

Comment:
You could declare a default config for all the calls and pass it over to functions or spread when needed

```js
const defaultConfig = { baseURL };

export const getCatalog = (env) =>
  networkClient.get(`/api/checks/catalog`, { ...defaultConfig, params: env });
```

Reply:
@CDimonaco i am going with config since the doc states `axios.get(url[, config])`